recipes/fisx/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "fisx" %}
+{% set version = "1.1.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c9dea8b6573e643cc966474a034fc6e916729fb9cba5cf361b5ec9a176dbc043
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  host:
+    - numpy
+    - pip
+    - python
+    - cython
+  run:
+    - numpy
+    - python
+
+test:
+  imports:
+    - fisx
+
+about:
+  home: https://github.com/vasole/fisx
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: Quantitative X-Ray Fluorescence Analysis Support Library
+  description: |
+          This software library implements formulas to calculate, given an experimental setup, the expected x-ray fluorescence intensities.
+          The library accounts for secondary and tertiary excitation, K, L and M shell emission lines and de-excitation cascade effects.
+          The basic implementation is written in C++ and a Python binding is provided.
+
+
```

Comment:
Please, remove this extra empty line here

Reply:
Done!